environments-and-requirements/environment-mac.yml

```diff
@@ -25,10 +25,11 @@ dependencies:
   - diffusers=0.6
   - einops=0.3
   - eventlet
-  - grpcio=1.46
   - flask=2.1
   - flask-socketio=5.3
   - flask-cors=3.0
+  - gfpgan==1.3.8
+  - grpcio=1.46
```

Comment:
I didn't see `grpcio` mentioned in the PR

Reply:
You were quite right! The change to `gfpgan` broke the CI tests. I'll push a commit that undoes those changes.

Reply:
> I didn't see `grpcio` mentioned in the PR

I reordered it to be in alphabetic order. It was already there.